frontend/src/app/shared/components/dynamic-forms/components/dynamic-form/dynamic-form.component.ts

```diff
@@ -285,6 +288,27 @@ export class DynamicFormComponent extends UntilDestroyedMixin implements OnChang
     return this._dynamicFormService.validateForm$(this.form, this.formEndpoint);
   }
 
+
+  handleCancel() {
+    if (this.form.dirty) {
+      this._confirmDialogService.confirm({
+        text: {
+          title: this._I18n.t('js.text_are_you_sure'),
+          text: this._I18n.t('js.text_data_lost'),
+        },
+      }).then(() => {
+        this.goBack();
+      })
+        .catch(() => {});
```

Comment:
You can remove the catch block

Reply:
With our new linting rules, there would be an error (sth like "unhandled promise"). So I'd argue to keep it 🤷‍♀️